### ISSUE TITLE
model.eval() when testing

### ISSUE THREAD

**Issue by enric1994** [cvlab-stonybrook/DM-Count#3]
I think you should add `model.eval()` in `test.py`. It won't alter the default result, but if batch normalization is enabled it will definitely affect the result. I had to change it because I am using ResNet50 instead of VGG19.

**Comment by Boyu-Wang** [comment]
Thanks. Updated.
